src/slack/slack.js

```diff
@@ -553,9 +553,8 @@ export class Slack {
     }
   }
 
-  async handleUserAction(stream, order, actionObj, user) {
-    const action = actionObj.name
-    const actionValue = actionObj.value
+  async handleUserAction(stream, order, action, user) {
+    const {name: actionName, value: actionValue} = action
 
     logger.info(`handling user action - name: ${action}, value: ${actionValue}, user: ${JSON.stringify(user)}, order: ${JSON.stringify(order)}`)
```

Comment:
please check that you have changed every occurrence of the renamed field, eg. you are suddenly logging the whole object instead of its name here, and that either just different from the original intention, or maybe even useless as it probably logs `[object Object]`

Reply:
@mekan-allaberdi the suggestions were just for your reference, in two of the three more is required on your side, please check what I wrote you in the comments

Reply:
@tamasbogi  @rtrembecky 
Here I just applied the suggestions and of course, fixed every occurrence of actionName and actionValue in the code.

Reply:
maybe you forgot to push your commits? or you're still working on it. you know

Reply:
@rtrembecky  
sure, did not push them yet.

Reply:
these kind of changes do as a single commit, as every one of them should be a working unit, you cannot just split renaming something on place and other to different commits